bindModel.js

```diff
@@ -61,13 +61,12 @@ var inputTypeBindings = {
       values.push(hasValue ? value : text)
 
       var selected = value === currentValue || text === currentValue
-
       if (selected) {
         selectedIndex = n
       }
 
       option.properties.selected = selected
-      option.properties.value = n
+      option.properties.value = typeof value === 'string' || value instanceof String ? value : n
```

Comment:
i reckon delete this whole line setting value, no need to set `value = n` at all if we use `selectedIndex` below.